plugin/pkg/scheduler/algorithm/priorities/util/non_zero.go

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
In fact it's impossible that there is a cycle - this was already there, so it can be left in that package.